rasa/cli/arguments/data.py

```diff
@@ -26,7 +25,6 @@ def set_convert_arguments(parser: argparse.ArgumentParser, data_type: Text):
 
     add_out_param(
         parser,
-        required=True,
```

Comment:
I think it still make sense *not* to convert the files in-place, and rather create converted copies using the `out` parameter.

Reply:
the reason why I did this change is not because of this, see [this comment](https://github.com/RasaHQ/rasa/pull/7183#discussion_r517883807)